### PR TITLE
Road to release

### PR DESCRIPTION
**Pull request by sivakumar-kailasam**
- Support diff highlighting & fixed issues on file name display in codeblocks.
- Added missing prism languages.
- FastBoot related changes
  - Remove prember to solve deployment issues due to OOM. (this is of course a temporary solution & can be reverted later)
  - Redirect `/current` to `/release` in the fastboot server.
  - Added heroku Procfile
  - Updated fastboot allowed hosts
  - Added ember-data-fastboot
  - Removed prember configuration
- Use components from latest ember-styleguide.
- Replace `http` with `https` in links inside app components.
- Removed legacy styles & repeated style definitions.
- Upgraded the app using `ember-cli-update`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,30 +29,21 @@ If you were hoping to help with the contents of the Guides instead of the App th
 
 You will need the following things properly installed on your computer.
 
-* [Git](https://git-scm.com/)
-* [Node.js](https://nodejs.org/) (with npm)
-* [Ember CLI](https://ember-cli.com/)
+- [Git](https://git-scm.com/)
+- [Node.js](https://nodejs.org/) (with npm)
+- [Ember CLI](https://ember-cli.com/)
```
Comment (jenweber) [inline README.md]: note to self - add yarn to this list

**Comment by jenweber** [comment]
Need to remove accidental commit of .gitkeep files and add them to the .gitignore

File: lib/content-guides-generator/index.js [hunk 2]
```diff
@@ -23,24 +20,6 @@ const VersionsSerializer = new Serializer('version', {
 
 const versions = yaml.safeLoad(readFileSync('node_modules/@ember/guides-source/versions.yml', 'utf8'));
 
-let premberVersions = [...versions.allVersions, 'release'];
```
Comment (jenweber) [inline lib/content-guides-generator/index.js]: why was this removed and what's the impact?

Comment (sivakumar-kailasam) [reply lib/content-guides-generator/index.js]: I didn't want this running every time, so I removed it. We can add it back when we bring back prember

File: package.json [hunk 3]
```diff
@@ -1,42 +1,51 @@
...
     "build-production": "ember build -e production",
-    "lint:js": "eslint ./*.js app config lib server tests",
+    "lint:js": "eslint ./*.js app blueprints config lib server tests",
     "start": "ember serve",
     "test": "ember test"
   },
-  "devDependencies": {
-    "@ember/guides-source": "github:ember-learn/guides-source#c18b8c5c3d9a404f509d64c187ff36e87b9ae374",
+  "dependencies": {
+    "@ember/guides-source": "github:ember-learn/guides-source#v3.1.4",
```
Comment (jenweber) [inline package.json]: why is the version locked down here? I'm curious to know what our deployment patterns should be as we move forward (not today)

**Comment by jenweber** [comment]
we don't need to fix it now, but this app needs more tests

File: package.json (hunk 3 above)
Comment (sivakumar-kailasam) [reply package.json]: This is a github release tag. Referring to commit ids can make it hard to go back and track that commit inside the source repo compared to release tags.

On deployments,
I'm hoping that whenever we create github releases/tags on guides-source, then may be via travis we can raise a PR for this repo.

**Comment by sivakumar-kailasam** [comment]
Addressed review comments with the last couple of commits.

**Comment by jenweber** [comment]
This should also be reviewed by @mansona before merging.

I looked through and things look ok!

**Comment by sivakumar-kailasam** [comment]
https://ember-guides-stage.herokuapp.com/release/ is the deployed app.

File: lib/content-guides-generator/index.js (hunk 2 above)
Comment (acorncom) [reply lib/content-guides-generator/index.js]: @sivakumar-kailasam I'm a bit puzzled about the removal of prember here as well. Could you discuss what prompted its removal here? Saw something briefly in the PR description, but didn't understand the reasoning behind it ...
